### PR TITLE
Add manual kubeconfig setup instructions

### PR DESCRIPTION
**Pull request by kiall**
Add instructions for manually adding a cluster to a pre-existing
kubeconfig file.

### CODE REVIEW

**Comment by kiall** [comment]
From codeclimate:

> Method done has 26 lines of code (exceeds 25 allowed). Consider refactoring. 

This check really needs to be removed, or at least set to a saner number.

**Comment by kiall** [comment]
image

Screenshot of the result

File: app/views/oidc/done.html.slim [hunk 1]
```diff
@@ -1,20 +1,50 @@
...
-p= link_to "Click here if that did not happen.", @redirect_target
-
 p
   | You can refer to it using kubectl by setting the <strong>KUBECONFIG</strong> environment variable,
   |  like <strong>KUBECONFIG=~/Downloads/kubeconfig kubectl get nodes</strong>.
 
 p
-  | You can also save it to your home in `~/.kube/config`, `kubectl` will automatically read this
-  |  file without the need to specify the <strong>KUBECONFIG</strong> environment variable.
+  | Alternativly, you can also save it to your home in `~/.kube/config`, `kubectl` will automatically
```
Comment (MaximilianMeister) [inline app/views/oidc/done.html.slim]: Alternatively ;-)

Comment (kiall) [reply app/views/oidc/done.html.slim]: Fixed - thanks!

**Comment by kiall** [comment]
@ereslibre / @MaximilianMeister / @vitoravelino - re review please? I've corrected the spelling mistake Max spotted.

**Pull request merged**